rust/operator/src/container.rs

```diff
@@ -56,6 +53,10 @@ use stackable_operator::{
 use std::{collections::BTreeMap, str::FromStr};
 use strum::{Display, EnumDiscriminants, IntoStaticStr};
 
+pub(crate) const KEYSTORE_DIR: &str = "/stackable/keystore";
+pub(crate) const KEYSTORE_VOLUME_NAME: &str = "keystore";
+pub(crate) const KEYSTORE_PASSWORD: &str = "changeit";
```

Comment:
NIT: `KEY_AND_TRUSTSTORE_PASSWORD` would be more expressive

Reply:
https://github.com/stackabletech/hdfs-operator/pull/392/commits/32dc916bc9b95727493d1c52f8d270dd962adf14